docs/DeveloperGuide.md

```diff
@@ -283,32 +283,165 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BobTheBodyBuilder` and the **Actor** is the `Gym Manager`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case:  Add a client**
```

Comment:
Can you add the use case id as well to all the use case descriptions so that if we ever need to do inclusion it'll be easier.
```suggestion
**Use case: UC01 - Add a client**
```

Reply:
I just added the use case numbers.